script/tool/lib/src/fetch_deps_command.dart

```diff
@@ -10,12 +11,14 @@ import 'common/package_looping_command.dart';
 import 'common/plugin_utils.dart';
 import 'common/repository_package.dart';
 
-/// Download dependencies for the following platforms {android}.
+const int _exitNothingRequested = 3;
+
+/// Download dependencies, both Dart and native.
 ///
 /// Specficially each platform runs:
 ///   Android: 'gradlew dependencies'.
-///   Dart: TBD (flutter/flutter/issues/130279)
-///   iOS: TBD (flutter/flutter/issues/130280)
+///   Dart: 'flutter pub get'.
+///   iOS/macOS: 'pod install'.
 ///
 /// See https://docs.gradle.org/6.4/userguide/core_dependency_management.html#sec:dependency-mgmt-in-gradle.
```

Comment:
nit: Consider linking to ios and dart documentation or removing android documentation.

Reply:
Since the others are just running commands that are run as part of building I couldn't think of any good docs to link to. I left Android because I think it's useful given that this isn't just us pre-running a command that's being directly run by `flutter`.